modules/card.js

```diff
@@ -1,92 +1,281 @@
 const rp = require("request-promise-native");
 const _ = require("lodash");
+const Discord = require("discord.js");
 const log = require("log4js").getLogger('card');
+const cheerio = require("cheerio");
 
 class MtgCardLoader {
     constructor() {
-        this.cardApi = "https://api.magicthegathering.io/v1/cards?name=";
-        this.commands = ["card"];
-        this.maxLength = 2000;
-        this.legalLimitations = ["Vintage","Legacy","Modern","Standard","Commander"];
+        this.commands = ["card", "price", "rulings", "ruling", "legal"];
+        this.cardApi = "https://api.scryfall.com/cards/search?q=";
+        this.manamojis = {
```

Comment:
Ah yes, this is where the magic happens. It turns out that Discord bots have the ability to use custom emojis globally on any server. So all I did here was upload the Manamojis to my Judgebot Discord server and link them from there in the code. I added some comments there now.